src/app/pages/Liquidity/components/modal/Withdraw/index.tsx

```diff
@@ -59,12 +62,11 @@ export const WithdrawModal: FC = () => {
       let type: WithdrawType | TokenSymbols = WithdrawType.IMBALANCE;
       if (Object.keys(tokenAmounts).length === 1) {
         type = Object.keys(tokenAmounts)[0] as TokenSymbols;
-      } else if (
-        Object.keys(tokenAmounts).length !== selectedPool.poolTokens.length
-      ) {
+      } else if (selectedToken === "mixed") {
         type = WithdrawType.IMBALANCE;
+      } else if (selectedToken === "combo") {
```

Comment:
consider holding this magic values `combo` and `mixed` into a `constant`
```suggestion
      } else if (selectedToken === tokenSelection.COMBO) {
```